testgrid/config.yaml

```diff
@@ -4378,6 +4382,11 @@ dashboards:
     base_options: 'exclude-filter-by-regex=^BeforeSuite$'
     alert_options:
       alert_mail_to_addresses: 'gke-kubernetes-accelerators-bugs@google.com'
+  - name: gce-stable2-stable1-gpu-upgrade
```

Comment:
Let's call this gce-1.9-1.10-gpu-upgrade

Reply:
done.